pkg/ocicni/ocicni.go

```diff
@@ -858,6 +858,12 @@ func buildCNIRuntimeConf(podNetwork *PodNetwork, ifName string, runtimeConfig Ru
 	if len(podNetwork.Aliases) > 0 {
 		rt.CapabilityArgs["aliases"] = podNetwork.Aliases
 	}
+
+	// set cgroupPath in Capabilities
+	if runtimeConfig.CgroupPath != "" {
+		rt.CapabilityArgs["cgroupPath"] = runtimeConfig.CgroupPath
```

Comment:
does this string exist anywhere we can reference directly?

Reply:
not that I see; I just followed the existing code that puts capability names here.

Reply:
Good suggestion for libcni sometime though.